src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="http://localhost:300api"
+      method="get"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label
+        class="form-field"
+        >
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            class="field"
+            required
+          >
+        </label>
+        <label
+          class="form-field"
+        >
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            class="field"
+            required
+          >
+        </label>
+        <label
+          class="form-field"
+        >
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            class="field"
+            required
+          >
+        </label>
+        <label
+          class="form-field"
+        >
+          Full date of birth:
+          <input
+            type="date"
+            name="date">
```

Comment:
closing bracket should appear on a new line in case if each of the attributes is on a new line
```suggestion
          <input
            type="date"
            name="date"
          >
```